llm_perf/benchmark_runners/update_llm_perf_cuda_pytorch.py

```diff
@@ -0,0 +1,126 @@
+from typing import Any, Dict, List
+
+from llm_perf.common.benchmark_runner import BenchmarkRunner
+from llm_perf.common.utils import GENERATE_KWARGS, INPUT_SHAPES
+from optimum_benchmark import PyTorchConfig
+from optimum_benchmark.benchmark.config import BenchmarkConfig
+from optimum_benchmark.launchers.process.config import ProcessConfig
+from optimum_benchmark.scenarios.inference.config import InferenceConfig
+
+
+class CUDAPyTorchBenchmarkRunner(BenchmarkRunner):
+    def __init__(self):
+        super().__init__(backend="pytorch", hardware="cuda")
```

Comment:
I would prefer `device="cuda"` to not confuse terminologies, optimum-benchmark already has a terminology and it's better to not make it ambiguous.